paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -664,8 +666,6 @@ internal class DefaultFlowController @Inject internal constructor(
 
         if (paymentResult is PaymentResult.Completed && selection != null && selection.isLink) {
             linkHandler.logOut()
-            // Clear the Link account state after logout to prevent stale credentials
-            linkAccountHolder.set(LinkAccountUpdate.Value(account = null))
```

Comment:
Why this change?

Reply:
That was added on this PR but noticed it was breaking certain flows, so reverted to just logout.